common/djangoapps/student/views.py

```diff
@@ -1740,6 +1747,7 @@ def create_account_with_params(request, params):
     if third_party_auth.is_enabled() and pipeline.running(request):
         running_pipeline = pipeline.get(request)
         third_party_provider = provider.Registry.get_from_pipeline(running_pipeline)
+        running_pipeline['kwargs']['data_sharing_consent'] = form.cleaned_data.get('data_sharing_consent', None)
```

Comment:
Please add a comment that this is for the `enterprise` app/feature. Otherwise someone looking at this might have no clue where to look for more info about what this line does.